_docs/_latest/setup/migrate.md

```diff
@@ -142,7 +142,7 @@ Import the existing resources to the Terraform state, replacing the
 uppercase values with the aforementioned values:
 
 ```sh
-./import.sh -m forseti -o ORG_ID -p PROJECT_ID -r RESOURCE_NAME_SUFFIX
+./import.sh -m MODULE_LOCAL_NAME -o ORG_ID -p PROJECT_ID -s RESOURCE_NAME_SUFFIX -z GCE_ZONE [-n NETWORK_PROJECT_ID]
```

Comment:
This change is not in the scope of this PR. Please remove this.

Reply:
This is a minor change to the syntax, I didn't want to create 2 PRs for a minor change.

Reply:
It's okay to open a PR for minor changes as well. This makes it easy to track changes when needed.

Reply:
Thanks for the suggestion, I agree that is best practice. Since these are minor, static changes for the website and not introducing new features, I am going to leave this as is and will consider this more for future PRs.